configs/alibaba-inc_clarity_design.json

```diff
@@ -14,9 +14,27 @@
       ]
     },
     {
-      "url": "https://at-design.alibaba-inc.com/doc/layout-pattern/",
```

Comment:
Please do not remove this URL as it is required to get a 200 and initiate the crawl

Reply:
@Mavericker-1996  could you provide us a compliant .xml sitemap to make sure we do not miss any content please?

Reply:
You can [find more details here](https://community.algolia.com/docsearch/tips.html#use-a-sitemapxml)